src/components/business/prevorders.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PrevOrderItem from "./prevorderitem.jsx";
+import axios from "axios";
+
+class PreviousOrder extends React.Component {
+  state = {
+    previousOrders: []
+  };
+
+  componentDidMount() {
+    this.getPreviousOrders();
+  }
+
+  getPreviousOrders() {
+    axios.get("http://localhost:7337/api/orders/complete").then(response => {
```

Comment:
I think we should swap out the localhost explicit call with an environmental variable so that this will work in production as well.